app/config/security.yml

```diff
@@ -1,10 +1,10 @@
 security:
     encoders:
-        TobiasOlry\TalklyBundle\Entity\User: plain
+        TobiasOlry\TalklyBundle\Entity\User: bcrypt
```

Comment:
Ok, symfony seems to use password_hash internally (http://stackoverflow.com/questions/25760520/does-symfony-derive-the-salt-from-the-hash-or-isnt-the-hash-salted-at-all) - but probably hardwires the hashing algorithm.